archetypes/oci-mp/src/main/resources/server/.dockerignore

```diff
@@ -0,0 +1 @@
+target/*
```

Comment:
Does this file really belong to our repository?

Reply:
Keeping this file.